ui/src/components/helpers/lists.js

```diff
@@ -0,0 +1,333 @@
+const accessTypeList = ["Unsigned", "Signed", "Alumni", "Admin", "Stewardship"];
+
+const lifestageList = [
+  "Building Blocks",
+  "Undergraduate",
+  "Postgraduate",
+  "Single Adult",
+  "Covenant",
+];
+
+const ministryTeamList = [
+  "Not Applicable",
+  "Intercessory Prayer Team",
+  "Creatives",
+  "Hospitality",
+  "Band",
+  "Audio/Visual",
+  "Creative Worship",
+  "Multimedia",
+  "Building Blocks",
+];
+
+const lifegroupList = [
+  "Not in a LIFE Group",
+  "BBB",
+  "DONKI",
+  "Drippin",
+  "Yeet Hay",
+  "GLOW UP",
+  "WASABAE",
+  "Refresh",
+  "GnT",
+  "SWAT",
+];
+
+const districtList = [
+  "Not Applicable",
+  "Islands",
+  "Kwai Tsing",
+  "North",
+  "Sai Kung",
+  "Sha Tin",
+  "Tai Po",
+  "Tsuen Wan",
+  "Tuen Mun",
+  "Yuen Long",
+  "Kowloon City",
+  "Kwun Tong",
+  "Sham Shui Po",
+  "Wong Tai Sin",
+  "Yau Tsim Mong",
+  "Central and Western",
+  "Eastern",
+  "Southern",
+  "Wan Chai",
+];
+
+const regionList = [
+  "Not Applicable",
+  "Kowloon",
+  "New Territories",
+  "Hong Kong Island",
+];
+
+const campusList = [
+  "Not Applicable",
+  "Hong Kong University of Science and Technology",
+  "The University of Hong Kong",
+  "The Chinese University of Hong Kong",
+  "The Hong Kong Polytechnic University",
+  "Hong Kong Baptist University",
+  "City University of Hong Kong",
+  "The Education University of Hong Kong",
+  "Lingnan University",
```

Comment:
Others with freetext?

Reply:
this is kinda annoying to allow, i think we currently allow people to just choose `Others`. Which I just added